docs/quick-ingestion-guides/tableau/configuration.md

```diff
@@ -60,7 +60,7 @@ If you do not see the Ingestion tab, please contact your DataHub admin to grant
 6. Select Tableau
 
 <p align="center">
-  <img width="70%" alt="Select Tableau from the options" src="https://raw.githubusercontent.com/datahub-project/static-assets-fork/main/imgs/guides/tableau/tableau-new-ingestion-source.png"/>
+  <img width="70%" alt="Select Tableau from the options" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/tableau/tableau-new-ingestion-source.png"/>
```

Comment:
looks like most of these image links are broken https://docs-website-1ua8gifbf-acryldata.vercel.app/docs/quick-ingestion-guides/powerbi/setup

Reply:
Hi @hsheth2 
Please verify again, I have corrected the links, however whenever from git I am trying to download or open the image, I am getting below error 

![image](https://user-images.githubusercontent.com/13194514/232372915-26fdec05-432c-4d28-a7fb-faee1e0c94d2.png)

The endpoint `raw.githubusercontent.com` is not rechable.